### DISCUSSION TITLE
Why columns widths are different on another computer?

### DISCUSSION THREAD

**Discussion by TaWReOn** [exceljs/exceljs#1684] [Q&A] [answered]
## 💬 Questions and Help

Hi all, 

I need to provide custom column width for my excel.  I'm achieving it with a `getColumn` method by setting a number of points to width property.
Main question is why on different computers I'm getting different column width both in pixels and points. Is it possible to have same value (at least in points) for all exported excels? For example the width of column is set to 16 points: on first computer I got **15.36 points** and **176px**, on the second - **15.29** points and **112px**.
Additional question: is it happening because of different monitor resolutions or because of excel settings?

Thanks in advance.

**Comment by Siemienik** [comment, marked as answer]
It is the MS Excel approach to store columns width in characters unit:

> The standard column width in Microsoft Excel 2000 is 8.43 characters; however, the actual width that you see on the screen varies, depending on the width of the font defined for the Normal style of your workbook. Changing the default font also changes the column width. This behavior occurs because of the way that Excel stores the column width information for individual fonts. This article discusses how column widths are determined.

Read more: https://docs.microsoft.com/en-us/office/troubleshoot/excel/determine-column-widths

BTW. While I was working on https://github.com/handsontable/spreadsheet-viewer I implemented a mechanism that maintains widths properly in a browser. It is much more complicated than it may look.
